### PR TITLE
Fixed Issue w/ FaceInteractable & Interaction text animation

### PR DESCRIPTION
**Pull request by chadleytan**
I tweaked the animation pop up of the interaction text to be a bit smoother and also fixed an issue w/ the FaceInteractable function where interacting with an interactable was only making player face one direction.

Also changed interaction to be called on Update rather than OnTriggerStay because Update is called much more frequently than OnTrigger Stay which becomes an issue when trying to interact with an interactable in rapid succession (although not necessarily what the player should be doing). 

Generally just some fixes and not any added functionality.

### CODE REVIEW

File: Assets/Scripts/Interactables/Interactable.cs [hunk 1]
```diff
@@ -27,6 +28,21 @@ protected void Update()
         {
             //TODO: Not sure if this works with 2 cameras?? Wait until later (using x negative scaling for now)
             interactTransform.LookAt(Camera.main.transform);
```
Comment (jurcaua) [inline Assets/Scripts/Interactables/Interactable.cs]: TODO: connect with the CameraManager i made to get the current camera transform

File: Assets/Scripts/Interactables/Interactable.cs [hunk 2]
```diff
@@ -27,6 +28,21 @@ protected void Update()
...
+
+            // Ensures player game object has been assigned
+            if (player is null)
+            {
+                Debug.Log("Player refernce not found");
```
Comment (jurcaua) [inline Assets/Scripts/Interactables/Interactable.cs]: Maybe change this to `Debug.LogError`
Also maybe this check should be in `FaceInteractable` not in the update method? (if so, try to do an early exit if `player is null` by returning early)

File: Assets/Scripts/Interactables/Interactable.cs [hunk 3]
```diff
@@ -82,23 +86,28 @@ public virtual void OnInteract()
...
-    public void FaceInteractable(GameObject player)
+    public void FaceInteractable()
     {
-        Vector3 dirToFace = transform.position + player.transform.position;
-        player.transform.rotation = Quaternion.Euler(0f, dirToFace.y, 0f);
+        Vector3 dirToFace = transform.position - player.transform.position;
+        dirToFace.y = 0f;
+
+        Quaternion rotation = Quaternion.LookRotation(dirToFace);
+        Debug.Log(rotation);
```
Comment (jurcaua) [inline Assets/Scripts/Interactables/Interactable.cs]: if we dont need to log this anymore, let's keep the console clean

**Review by jurcaua — CHANGES_REQUESTED** [review]
Small comments but otherwise wonderful stuff!

File: Assets/Scripts/Interactables/Interactable.cs (hunk 1 above)
Comment (chadleytan) [reply Assets/Scripts/Interactables/Interactable.cs]: Was able to connect it to the CameraManager!

**Review by jurcaua — APPROVED** [review]
a beauty indeed

**Pull request merged**